flimfit.html

```diff
@@ -57,6 +57,8 @@
 
 <p>A list of file formats supported by FLIMfit can be found on the OME website <a href="http://www.openmicroscopy.org/site/support/partner/flimfit/flimfit-supported-file-formats" target="_blank" class="external-url">FLIMfit page</a>.</p>
 
+<p>further details about FLIMfit can be found on the <a href="http://imperial-photonics.github.io/FLIMfit/" target="_blank" class="external-url">FLIMfit website</a>.</p>
```

Comment:
f -> F, or make this into one sentence with the previous line, otherwise this looks fine on staging.